docs/components/connectors/out-of-the-box-connectors/email.md

```diff
@@ -126,6 +126,10 @@ The task returns a JSON object containing detailed information about the email:
 - `attachments`: A list of all the email's attachments, provided as a document reference.
 - `receivedDateTime`: The email's reception datetime
 
+:::note
+Starting from version 8.7.0, the email Connector supports storing the response as a document. See additional details and limitations in [document handling](/components/concepts/document-handling.md).
+:::
+
```

Comment:
"... the email Connector supports sending Camunda documents as attachments, or storing incoming attachments as Camunda documents"